Configuration/Geometry/python/GeometryExtended2023Reco_cff.py

```diff
@@ -15,13 +15,12 @@
 #Muon
 from Geometry.MuonNumbering.muonNumberingInitialization_cfi import *
 from RecoMuon.DetLayers.muonDetLayerGeometry_cfi import *
-from Geometry.GEMGeometry.gemGeometry_cfi import *
```

Comment:
Just to clarify, is Extended2023 supposed to have ME0? If not, it shouldn't be added.